tests/test_hubspot_bookmarks.py

```diff
@@ -53,6 +53,9 @@ def create_test_data(self, expected_streams):
                                  for stream in expected_streams}
 
         for stream in expected_streams - {'contacts_by_company'}:
+            """ Create custom properties for contacts """
```

Comment:
Is this meant to be a comment instead of a doc string?

Reply:
Meant to be a comment. I changed to #, don't really know the difference